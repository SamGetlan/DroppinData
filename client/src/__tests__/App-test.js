import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/App.jsx';
import { MemoryRouter } from 'react-router';

test('App renders correctly', () => {
  const element = renderer.create(
    <MemoryRouter>
      <App />
    </MemoryRouter>
  );
  let tree = element.toJSON();
  expect(tree).toMatchSnapshot();
});