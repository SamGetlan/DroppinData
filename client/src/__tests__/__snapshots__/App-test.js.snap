// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  id="app"
>
  <div
    className="navbar row"
  >
    <a
      className="navButtonContainer"
      href="/stats/dashboard"
      onClick={[Function]}
    >
      <button
        className="navButton col-2 col-s-6 stats"
        onClick={[Function]}
      >
        Stats
      </button>
    </a>
    <a
      className="navButtonContainer"
      href="/home/map"
      onClick={[Function]}
    >
      <button
        className="navButton col-2 col-s-6 map"
        onClick={[Function]}
      >
        Full Map
      </button>
    </a>
    <a
      className="navButtonContainer"
      href="/home/filterLocations"
      onClick={[Function]}
    >
      <button
        className="navButton col-2 col-s-6 filter"
        onClick={[Function]}
      >
        Filter Locations
      </button>
    </a>
    <a
      className="navButtonContainer"
      href="/home/login"
      onClick={[Function]}
    >
      <button
        className="navButton col-3 col-s-6 login"
        onClick={[Function]}
      >
        Sign Up or Login
      </button>
    </a>
  </div>
  <div>
    <div
      id="actionButtonContainer"
    >
      <button
        id="actionButton"
        onClick={[Function]}
      >
        Where We Droppin'
      </button>
    </div>
    <div
      id="backSplashContainerWrapper"
    >
      <div
        id="backSplashContainer"
      >
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Chair
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/d8.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Loot Lake House
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/e4.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            West of Loot Lake
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/d4.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            North of Loot Lake
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/e3.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Prison
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/h8.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Shipping Containers
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/h4.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Lucky Landing
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/f9.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Flush Factory
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/d9.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Tilted Towers
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/d5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Shifty Shafts
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/d7.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Haunted Hills
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/b2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Anarchy Acres
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/f2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Lonely Lodge
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/i5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Wailing Woods
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/i3.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Moisty Mire
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/i9.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Fatal Fields
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/f8.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Greasy Grove
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/c7.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Snobby Shores
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/b5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Junk Junction
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/b2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Pleasant Park
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/c3.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Dusty Depot
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/f5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Salty Springs
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/f7.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Retail Row
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/h6.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Tomato Town
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/g4.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            House on the Hill
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/e8.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Town west of Motel
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/c2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Town northeast of Flush Factory
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/e9.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Buildings West of Tilted Towers
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/c5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Buildings North of the Prison
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/i7.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Motel
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/d2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Risky Reels
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/h2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Mansion South of Lonely Lodge
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/i5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Dusty Divot
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/f5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Buildings East of Junk Junction
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/c2.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Silo in Mountain Northeast of Snobby Shores
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/b4.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
        <div
          className="tile"
          onClick={[Function]}
        >
          <p
            className="tileTitle"
          >
            Trailer Park
          </p>
          <img
            alt="map view of location"
            className="mapImage"
            height="80%"
            src="/locationPics/i5.jpg"
            width="100%"
          />
          <div
            className="tileStats"
          >
            <span
              className="tileAvgPlace tileStat"
            >
              Avg. Place: 
              N/A
            </span>
            <span
              className="tileAvgKills tileStat"
            >
              Avg. Kills: 
              N/A
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    id="userFormContainerWrapper"
  >
    <div
      className="mobile-85"
      id="userFormContainer"
    >
      <div
        id="userFormBox"
      >
        <div
          id="userFormOptionsBar"
        >
          <button
            className="userFormOptionsButton"
            id="loginUserFormOptionButton"
            onClick={[Function]}
          >
            Login
          </button>
          <button
            className="userFormOptionsButton"
            id="signUpUserFormOptionButton"
            onClick={[Function]}
          >
            Sign Up
          </button>
        </div>
        <div
          id="userFormMain"
        >
          <h2
            id="userFormTitle"
          >
            Log In
          </h2>
          <h4
            id="userFormSubTitle"
          >
            Please fill in your username and password to login
          </h4>
          <a
            href="/home/accountRecovery"
            onClick={[Function]}
          >
            <p
              id="forgotPasswordLink"
            >
              Forgot username/password?
            </p>
          </a>
          <form
            id="signUpForm"
          >
            <label
              htmlFor="usernameInput"
            >
              Username 
            </label>
            <input
              id="usernameInput"
              placeholder="Enter Username"
              type="text"
            />
            <label
              htmlFor="passwordInput"
            >
              Password 
            </label>
            <input
              id="passwordInput"
              placeholder="Enter Password"
              type="password"
            />
          </form>
          <button
            id="submitSignUpForm"
            onClick={[Function]}
          >
            Log In
          </button>
        </div>
      </div>
      <a
        href="/home"
        id="closeButtonContainer"
        onClick={[Function]}
      >
        <button
          className="closeButton"
          onClick={[Function]}
        >
          Close
        </button>
      </a>
    </div>
  </div>
</div>
`;
